frontend/src/components/Registration/Registration.test.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import Registration from './Registration';
+import { fireEvent, render, screen, waitForElement } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import { Provider } from 'react-redux';
+import { store } from '../../app/store';
+import BaseService from '../../app/baseService';
+
+jest.mock('../../app/baseService', () => jest.fn());
+
+describe('Registration', () => {
+   jest.setTimeout(7000);
+   // it('renders without error', () => {
+   //    render(
+   //       <Provider store={store}>
+   //          <Registration />)
+   //       </Provider>
+   //    );
+
+   //    const fnameInput = screen.getByTestId('r-fname');
+   //    const lnameInput = screen.getByTestId('r-lname');
+   //    const emailinput = screen.getByTestId('r-email');
+   //    const passwordInput = screen.getByTestId('r-password');
+   //    const cpasswordInput = screen.getByTestId('r-cpassword');
+   //    const button = screen.getByTestId('r-button');
+
+   //    expect(fnameInput).toHaveTextContent('Name');
+   //    expect(lnameInput).toHaveTextContent('Surname');
+   //    expect(emailinput).toHaveTextContent('Email Address');
+   //    expect(passwordInput).toBeInTheDocument();
+   //    expect(cpasswordInput).toBeInTheDocument();
+   //    expect(button ).toBeInTheDocument();
+   // });
```

Comment:
Leaving commented code doesn't look good...

Reply:
When I was working on the tests it was throwing me a bug at one point and I didn't know which test it was throwing for, so I commented code on the good one... and forgot to undo it. My mistake. 🙈